server/src/main/java/org/eclipse/lsp/cobol/core/engine/CobolLanguageEngine.java

```diff
@@ -126,9 +132,13 @@ public CobolLanguageEngine(
     CobolParser.StartRuleContext tree = parser.startRule();
     timingBuilder.getParserTimer().stop();
 
+    timingBuilder.getSplittingLanguageTimer().start();
+    Map<Token, EmbeddedCode> embeddedCodeParts = extractEmbeddedCode(listener, tree);
```

Comment:
Is it embedded SQL code now? If yes, I guess we will have for CICS in future. 
Isn't it better to use readable name - embeddedSqlCodeParts.